hack/download-charts.sh

```diff
@@ -1,34 +1,45 @@
 #!/usr/bin/env bash
 
-set -e -u -o pipefail
+set -e -x -u -o pipefail
 
 : "${MAISTRA_RELEASE_STREAM:=$1}"
 : "${ISTIO_REPO:=$2}"
 : "${ISTIO_COMMIT:=$3}"
 
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 REPO_ROOT=$(dirname "${SCRIPT_DIR}")
-HELM_DIR="${REPO_ROOT}/resources/charts/${MAISTRA_RELEASE_STREAM}"
+MANIFEST_DIR="${REPO_ROOT}/resources/${MAISTRA_RELEASE_STREAM}"
+CHARTS_DIR="${MANIFEST_DIR}/charts"
+PROFILES_DIR="${MANIFEST_DIR}/profiles"
 CONFIG_DIR="${REPO_ROOT}/config"
 
 ISTIO_FILE="${ISTIO_COMMIT}.tar.gz"
 ISTIO_URL="${ISTIO_REPO}/archive/${ISTIO_COMMIT}.tar.gz"
 WORK_DIR=$(mktemp -d)
 EXTRACT_DIR="${ISTIO_REPO##*/}-${ISTIO_COMMIT}"
-EXTRACT_CMD="tar zxf ${ISTIO_FILE} ${EXTRACT_DIR}/manifests/charts ${EXTRACT_DIR}/manifests/addons/dashboards"
+EXTRACT_CMD="tar zxf ${ISTIO_FILE} ${EXTRACT_DIR}/manifests/charts ${EXTRACT_DIR}/manifests/profiles ${EXTRACT_DIR}/manifests/addons/dashboards"
 
-function downloadIstioCharts() {
-  rm -rf "${HELM_DIR}"
-  mkdir -p "${HELM_DIR}"
+YQ="go run github.com/mikefarah/yq/v4"
```

Comment:
YQ is already present is the istio upstream builder image. We should switch to it ASAP. @bmangoen is working on this.

Reply:
I'll put this PR on hold until @bmangoen's PR is merged.